cabot/rest_urls.py

```diff
@@ -124,33 +125,32 @@ class ViewSet(viewset_class):
     ),
 ))
 
-'''
-Omitting user API, could expose/allow modifying dangerous fields.
+# User API is off by default, could expose/allow modifying dangerous fields
+if settings.EXPOSE_USER_API:
+    router.register(r'users', create_viewset(
+        arg_model=django_models.User,
+        arg_fields=(
+            'password',
+            'is_active',
+            'groups',
+            #'user_permissions', # Doesn't work, removing for now
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+        ),
+    ))
 
-router.register(r'users', create_viewset(
-    arg_model=django_models.User,
-    arg_fields=(
-        'password',
-        'is_active',
-        'groups',
-        #'user_permissions', # Doesn't work, removing for now
-        'username',
-        'first_name',
-        'last_name',
-        'email',
-    ),
-))
+    router.register(r'user_profiles', create_viewset(
+        arg_model=models.UserProfile,
+        arg_fields=(
+            'user',
+            'mobile_number',
+            'hipchat_alias',
```

Comment:
Poking around a bit, trying to see what might go wrong... 

I don't see plugins adding UserProfile fields, so I don't think we'd need to update arg_fields here. 
I think they just add AlertPluginUserData.

Anyway, let me know if there's anything I can do to address the concern.